snake/graphics.py

```diff
@@ -12,10 +12,15 @@ def drawTile(x, y, tile='', color=None):
 
     x = x * 2 + stage.padding[3] * 2 + stage.width / 2
     y += stage.padding[0] + stage.height / 2
-
-    screen.addstr(y, x, tile, color)
+    try:
+        screen.addstr(y, x, tile, color)
+    except curses.error:
+        pass
```

Comment:
The code fix only allows `curses.error` to pass, all other errors will still be visible.